cipher/src/stream_core.rs

```diff
@@ -195,7 +195,7 @@ impl_counter! { u32 u64 u128 }
 /// In case if `N` is less or equal to 1, buffer of arrays has length
 /// of zero and tail is equal to `self`.
 #[inline]
-fn into_chunks<T, N: ArrayLength<T>>(buf: &mut [T]) -> (&mut [GenericArray<T, N>], &mut [T]) {
+fn into_chunks<T, N: ArraySize>(buf: &mut [T]) -> (&mut [Array<T, N>], &mut [T]) {
```

Comment:
Maybe we should move this method into `hybrid-array`? I think we had a recent discussion about it.

Reply:
Sure, although perhaps that could be a followup?